app/controllers/admin/decisions_controller.rb

```diff
@@ -5,18 +5,17 @@ def new
   end
 
   def create
-    @split = Split.find params[:split_id]
-    Split.transaction do # FIXME: This makes sure this controller either succeeds or fails atomically until we can async the work
-      @decision = @split.create_decision!(create_params)
-    end
-    flash[:success] = "Reassigned #{@decision.count} visitors to #{@decision.variant}"
-    redirect_to admin_split_path(@split)
+    split = Split.find params[:split_id]
+    decision = split.build_decision(create_params)
+    CreateDecisionJob.perform_later(split, create_params)
+    flash[:success] = "Queued decision to reassign #{decision.count} visitors to #{decision.variant}"
+    redirect_to admin_split_path(split)
   end
```

Comment:
in this controller action we have to build the decision in order to provide the same level of feedback to the admin performing the decision. we end up throwing that decision away because we can't serialize an ActiveModel in ActiveJob. At least, it wasn't immediately obvious how to accomplish that.